### PR TITLE
Update charts to v2.1 for addition of CT tests

### PR DESCRIPTION
**Pull request by mbuchmann-hpe**
## Summary and Scope

Updated charts to v2.1 and added CT Smoke Tests
SCSD does not have any CT functional tests

## Issues and Related PRs

CASMHMS-5420

### CODE REVIEW

**Review by shunr-hpe — APPROVED** [review]

File: charts/v2.1/cray-hms-scsd/tests/test-smoke.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    spec:
+      restartPolicy: Never
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
+        runAsGroup: 65534
+      containers:
+        - name: "smoke"
+          image: "{{ .Values.tests.image.repository }}:{{ .Values.global.testVersion }}"
```
Comment (schooler-hpe) [inline charts/v2.1/cray-hms-scsd/tests/test-smoke.yaml:30]: Did you try deploying the new chart and running the smoke tests on a live system? From what I can tell, the variables in this smoke container section have not been defined in values.yaml so the tests won't run.

File: changelog/v2.1.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Changelog for v2.0
```
Comment (schooler-hpe) [inline changelog/v2.1.md]: I think this should be v2.1.

File: changelog/v2.1.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+All notable changes to this project for v2.0.Z will be documented in this file.
```
Comment (schooler-hpe) [inline changelog/v2.1.md:3]: Same here.

File: changelog/v2.1.md (hunk 2 above)
Comment (mbuchmann-hpe) [reply changelog/v2.1.md]: Updated

File: changelog/v2.1.md (hunk 3 above)
Comment (mbuchmann-hpe) [reply changelog/v2.1.md:3]: Updated

File: charts/v2.1/cray-hms-scsd/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: "cray-hms-scsd"
+version: 2.0.2
```
Comment (schooler-hpe) [inline charts/v2.1/cray-hms-scsd/Chart.yaml]: This needs to be updated to version: 2.1.0.

**Review by schooler-hpe — DISMISSED** [review]

File: charts/v2.1/cray-hms-scsd/Chart.yaml (hunk 4 above)
Comment (mbuchmann-hpe) [reply charts/v2.1/cray-hms-scsd/Chart.yaml]: Updated

**Review by alvarez3-hpe — APPROVED** [review]

**Pull request merged**